syntax = "proto3";
import "google/protobuf/timestamp.proto";

message LocationMessage {
  int32 person_id = 1;
  double latitude = 2;
  double longitude = 3;
  google.protobuf.Timestamp creation_time = 4;
}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
}
